src/main-page.ts

```diff
@@ -38,22 +40,36 @@ class MainPage implements Page {
         this._recordButton?.classList.remove("d-none");
     }
 
-    private onClickModalReady() {
-        this._calibrationModal?.hide();
-        this._recordButton?.classList.remove("d-none");
-
+    private async onClickModalReady() {
         if (this._orientation.lastOrientation) {
             const initialOrientation = {
                 alpha: this._orientation.lastOrientation.alpha ?? 0,
                 beta: this._orientation.lastOrientation.beta ?? 0,
                 gamma: this._orientation.lastOrientation.gamma ?? 0
             }
+            console.log(initialOrientation);
```

Comment:
do you really need this logging?

Reply:
I will delete it after I finish with data management